rollup.config.js

```diff
@@ -51,9 +52,88 @@ const plugins = (es5) => [
     terser({
         toplevel: true,
         compress: {
-            // 5 is the default if unspecified
             ecma: es5 ? 5 : 6,
         },
+        mangle: es5
+            ? undefined // Don't mangle properties in the es5 build, as it relies on helpers with don't work well with mangling.
+            : {
+                  // Note:
+                  // PROPERTY MANGLING CAN BREAK YOUR CODE
```

Comment:
Without knowing much about how this works it would be super useful if we could let people know somehow that they've written a function that will be mangled (and possibly make them safe list it). It feels like that's the best way to protect against peoples code being minified unintendedly (not sure that's a word)

Reply:
The latter would be great but probably not necessary if it is too much hassle. I just worry that people don't necessarily see the README and end up dealing with the consequences

Reply:
Maybe even an eslint rule would suffice if there's a prewritten one

Reply:
> The latter would be great
Done! :)